github/repos_contents_test.go

```diff
@@ -91,14 +91,14 @@ func TestRepositoriesService_GetReadme(t *testing.T) {
 }
 
 func TestRepositoriesService_DownloadContents_Success(t *testing.T) {
-	setup()
+	serverBaseURL, mux, client, teardown := setup()
```

Comment:
Minor, but I'd rename `serverBaseURL` to just `serverURL` for consistency with the return variable name in `setup`. At least I don't see a good reason to give it a different, more verbose name.